signal-hook-tokio-0_1/src/lib.rs

```diff
@@ -0,0 +1,118 @@
+#![cfg(not(windows))]
+//! A module for integrating signal handling with the tokio runtime.
+//!
+//! This provides the [`Signals`](struct.Signals.html) struct as an abstraction which can be
+//! turned into a [`Stream`](https://docs.rs/tokio/0.1.22/tokio/prelude/trait.Stream.html) of
+//! signals.
+//!
+//! An example showing how to use it can be found in the documentation of the
+//! [`into_stream`](struct.Signals.html#method.into_stream) method.
```

Comment:
Yeah, there is no real reason left to have the into_stream method. I'll change this to a simpler approach.

Reply:
There is no real reason why this are still two structs. I thought it may be nice to add an async version of a wait() method to the Signal struct which returns a Future<Pending> to mimic the blocking Signal struct. But the legacy Tokio and Future crates didn't allow for a nice usage with their future combinators style. So I dropped this method but didn't think about merging both structs.

 I'll change it so Stream is directly implemented for the Signals struct.